### DISCUSSION TITLE
App\Filament\Resources\UserResource::App\Filament\Resources\{closure}(): Argument #1 ($livewire) must be of type App\Filament\Resources\Page, App\Filament\Resources\UserResource\Pages\CreateUser given, called in /var/www/html/vendor/filament/support/src/Concerns/EvaluatesClosures.php on line 35

### DISCUSSION THREAD

**Discussion by gustiwdr** [filamentphp/filament#13282] [Help]
### Package

Form builder

### Package Version

v3.2.91

### How can we help you?

So I use this for the 'password' section in my forms (Resource\UserResource):
```->required(fn (Page $livewire): bool => $livewire instanceof CreateRecord)```

but the error message appears as I wrote in the title

**Comment by dissto** [comment]
https://filamentphp.com/docs/3.x/forms/advanced#injecting-the-current-form-operation ?

**Comment by HabibiAlfahrezi** [comment]
Add this on top
use Filament\Pages\Page;
use Filament\Resources\Pages\CreateRecord;

**Reply by HabibiAlfahrezi** [reply]
That should work
